android/java/res/layout/brave_rewards_tip_confirmation.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@android:color/white"
+    tools:context=".BraveRewardsTipConfirmationFragment">
+
+    <TextView
+        android:id="@+id/id_thanks_for_support"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/poppins_medium"
+        android:text="@string/thanks_for_support"
+        android:textStyle="bold"
+        android:textColor="#212529"
```

Comment:
we should avoid using hard coded color codes. i think we have  this color code for text for other parts

Reply:
removed all hard coded color code